<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Office</class>
 <widget class="QDialog" name="Form_Office">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>103</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Office Code:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>103</y>
     <width>552</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>227</y>
     <width>103</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Postal Code:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>196</y>
     <width>552</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>134</y>
     <width>103</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Address Line 2:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_8">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>227</y>
     <width>552</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>103</y>
     <width>103</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Address Line 1:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_9">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>258</y>
     <width>552</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>165</y>
     <width>103</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>State:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>10</y>
     <width>552</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>41</y>
     <width>552</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>72</y>
     <width>552</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="process_office_record">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>289</y>
     <width>552</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>null</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>72</y>
     <width>103</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>196</y>
     <width>103</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Country:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>134</y>
     <width>552</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>165</y>
     <width>552</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>258</y>
     <width>103</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Territory:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>41</y>
     <width>103</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>City:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>lineEdit_6</tabstop>
  <tabstop>lineEdit_7</tabstop>
  <tabstop>lineEdit_8</tabstop>
  <tabstop>lineEdit_9</tabstop>
  <tabstop>process_office_record</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
